pkg/env/sensor.go

```diff
@@ -48,4 +48,7 @@ var (
 
 	// DeduperStateSyncTimeout defines the maximum time Sensor will wait for the expected deduper state coming from Central
 	DeduperStateSyncTimeout = registerDurationSetting("ROX_DEDUPER_STATE_TIMEOUT", 30*time.Second)
+
+	// ProcessIndicatorBufferSize indicates how many process indicators will be kept in Sensor while offline.
+	ProcessIndicatorBufferSize = RegisterIntegerSetting("ROX_SENSOR_PROCESS_INDICATOR_OFFLINE_BUFFER_SIZE", 100)
```

Comment:
I omitted `OFFLINE` from my FF name, only named it buffer size. Technically the buffer is still there if the detector takes too long to pick up the messages. 

I don't have a strong opinion, and if you prefer to keep the `OFFLINE` in the name, I'll change mine. I just think that the env names should be similar.

Reply:
> I just think that the env names should be similar.

I agree. I've changed it to omit `OFFLINE`.